components/layout/PlayCard.tsx

```diff
@@ -0,0 +1,90 @@
+import ArrowForwardIcon from '@mui/icons-material/ArrowForward';
+import {
+  Box,
+  Button,
+  Card, CardContent, Grid, Typography,
+} from '@mui/material';
+import React from 'react';
+
+import ImageContainer from './ImageContainer.tsx';
+
+interface PlayCardProps {
+  title?: string;
+  details: string;
+  iconSource?: string;
+  ctaButton?: string;
+  textColor: string;
+  backgroundColor: string;
+  cardWidth: number,
+  cardHeight: number,
+}
+
+export default function PlayCard({
+  title, details, iconSource, ctaButton, textColor, backgroundColor, cardWidth, cardHeight,
+}: PlayCardProps) {
+  return (
+    <Card
+      sx={{
+        width: cardWidth,
+        minHeight: cardHeight,
+        bgcolor: `${backgroundColor}`,
+        color: `${textColor}`,
+        display: 'flex',
+        flexDirection: 'column',
+        px: 1,
+      }}
+    >
+      <CardContent sx={{ flexGrow: 1, display: 'flex', flexDirection: 'column' }}>
+        <Grid container direction="column" justifyContent="space-between" sx={{ flexGrow: 1 }}>
+          <Grid item sx={{ mt: 4 }}>
+            <Typography variant="overline" sx={{ fontWeight: 700, textTransform: 'none' }}>{title}</Typography>
+          </Grid>
+          <Grid item sx={{ mt: 4 }}>
+            <Typography sx={{ lineHeight: '21.79px' }} variant="body2">{details}</Typography>
+          </Grid>
+          {iconSource && (
+            <Grid item sx={{ mt: 4 }}>
+              <ImageContainer
+                src={iconSource}
+                width={200}
+                height={120}
+                alt=""
+                style={{ maxWidth: '100%', height: 'auto' }}
+                useImageDimensions
+              />
+            </Grid>
+          )}
+          <Grid container sx={{ mt: 'auto' }}>
+            <Grid item>
+              <Button
+                href="/calculator/head-initial-1-cont"
```

Comment:
We should abstract the href, color, className, and ariaLabel so that this card can be reused without these hardcoded values.